network/src/protocols/identify/mod.rs

```diff
@@ -541,7 +538,11 @@ impl Callback for IdentifyCallback {
         MisbehaveResult::Continue
     }
 
-    fn misbehave(&mut self, _peer_id: &PeerId, _kind: Misbehavior) -> MisbehaveResult {
+    fn misbehave(&mut self, session: &SessionContext, reason: Misbehavior) -> MisbehaveResult {
+        error!(
+            "IdentifyProtocol detect misbehave, session: {:?}, reason: {:?}",
```

Comment:
```suggestion
            "IdentifyProtocol detects abnormal behavior, session: {:?}, reason: {:?}",
```

Reply:
I corrected the sentences as you suggested in the last force-push, thanks.